docs: RETURNS STATUS AS ROW (#4656)
yugabyte/yugabyte-db: docs/content/latest/api/ycql/dml_delete.md

(1 rows)
```

### RETURNS STATUS AS ROW

When executing a batch in YCQL, the protocol just allows returning one error/return status.  

> The `RETURNS STATUS AS ROW` feature addresses this limitation and a status row for each statement.

Reviewer: something is missing in this sentence.

> When executing a batch in YCQL, the protocol just allows returning one error/return status.  
> The `RETURNS STATUS AS ROW` feature addresses this limitation and a status row for each statement.

Reviewer 2: ```suggestion
When executing a batch in YCQL, the protocol returns only one error or return status.  
The `RETURNS STATUS AS ROW` feature addresses this limitation and adds a status row for each statement.
```
